lib/monitor.js

```diff
@@ -134,8 +139,18 @@ internals.Monitor.prototype.start = function (callback) {
                 item.reporter = require(item.reporter);
             }
 
-            Hoek.assert(typeof item.reporter === 'function', 'reporter key must be a constructor function');
-            Hoek.assert(typeof item.events === 'object', 'reporter must specify events to filter on');
+            // attempt to upgrade hint to reporter name
+            var reachOptions = {
+                functions: true
+            };
+            var reporterName = Hoek.reach(item, 'reporter.attributes.name', reachOptions);
+            reporterName = reporterName || Hoek.reach(item, 'reporter.attributes.pkg.name', reachOptions);
+            if (typeof reporterName === 'string' && reporterName.length > 0) {
```

Comment:
I think just a truth check here would be fine. If you're going to be silly and have your reporter `name` be `function () {}` then you don't deserve to see a nice name during the error phase.